apps/condo/domains/ticket/access/TicketFilterTemplate.js

```diff
@@ -13,7 +13,7 @@ async function canReadTicketFilterTemplates ({ authentication: { item: user } })
     if (user.isAdmin) return {}
 
     return {
-        employee: { organization: { ...queryOrganizationEmployeeFor(user.id) } },
```

Comment:
1) Is permission check necessary here?
2) If necessary, is it necessary to check this right when creating/updating?

Reply:
added checks, updated tests